src/controllers/shared.controllers/director.correspondence.address.manual.controller.ts

```diff
@@ -171,12 +173,13 @@ export const buildValidationErrors = (validationStatusResponse: ValidationStatus
   return validationErrors;
 }
 
-export const renderPage = (req: Request, res: Response, serviceAddress: Address, officerFiling: OfficerFiling, validationErrors: ValidationError[], templateName: string, backUrlPath: string) => {
+export const renderPage = async (req: Request, res: Response, serviceAddress: Address, officerFiling: OfficerFiling, validationErrors: ValidationError[], templateName: string, backUrlPath: string, isUpdate: boolean, session: Session) => {
```

Comment:
I'm not quite sure why the sonar has not kicked off build but there's going to be a code smell for number of method params. 
I think you can create a new interface like [renderPageParam](https://github.com/companieshouse/officer-filing-web/blob/main/src/utils/renderArrayPageParams.ts) for your classes to get around it.

Reply:
Ah I didn't realise there would a smell for it, will address this now